Sources/CreateProject/CreateProject.swift

```diff
@@ -21,15 +21,20 @@ private struct CreateProject {
                 throw ChangeDirectoryError(path: projectPath)
             }
 
-            let projectName = try getProjectName()
-
             let contents = try fileManager.contentsOfDirectory(atPath: fileManager.currentDirectoryPath)
             guard contents.isEmpty else {
                 print(color: .red, "Directory at \(projectPath) must be empty, but found: \(contents.joined(separator: ", "))")
                 exit(1)
             }
 
+            let projectName = try getProjectName()
+            
             let executableName = try getExecutableName()
+            
+            if projectName == executableName {
+                print(color: .red, "Project name and Executable Name must be different")
+                exit(0)
```

Comment:
I think this should exit with non-zero (1) as this is an error.
```suggestion
                exit(1)
```